pkg/transferer/transferer.go

```diff
@@ -50,6 +50,19 @@ func (t Transferer) readChunksFromSource(ctx context.Context, p ChunkPool, chunk
 			log.Debug().Msg("Context is done, stopping chunks reading")
 			return ctx.Err()
 		default:
+			switch lc.GetLatestStatus() {
+			case LoadStatusWait:
+				time.Sleep(time.Second * 10) // TODO: make duration configurable
+				log.Debug().Msgf("Got wait load status: putting chunks reading to sleep for %d seconds", 10)
+				continue
+			case LoadStatusNone:
+				time.Sleep(time.Second * 2) // TODO: make duration configurable
+				log.Debug().Msgf("Got none load status: putting chunks reading to sleep for %d seconds", 2)
+				continue
+			case LoadStatusTerminate:
+				log.Debug().Msg("Got terminate load status: stopping chunks reading")
+				return nil
```

Comment:
and in this case, we must return error, because no valid dump is created

Reply:
resloved in 44c38c52f5f762ad3262986f5032da015d65fb52